docs/conf/portland/2025/news/welcome-portland-2025.md

```diff
@@ -0,0 +1,112 @@
+
+<p style="margin: 2em 0;">
+<table border="0" cellpadding="0" cellspacing="0" style="background-color:#2ECC71; border:1px solid #4a4a4a; border-radius:5px;">
+<tr>
+  <td align="center" valign="middle" style="color:#FFFFFF; font-family:Helvetica, Arial, sans-serif; font-size:16px; font-weight:bold; letter-spacing:-.5px; line-height:150%; padding-top:15px; padding-right:30px; padding-bottom:15px; padding-left:30px;">
+     <a href="https://docs.google.com/spreadsheets/d/1Yyo2V5Xzwz7KhF4xkEmGNFNHld7TEUG7VfitMzM_Ohc/edit?usp=sharing" target="_blank" style="color:#FFFFFF; text-decoration:none; border-bottom: none;">Sign up to Lead a Monday Unconference</a>
+  </td>
+</tr>
+</table>
+</p>
+
+For those who are new to Unconferences or nervous about what makes a "good" topic, we have a vast range of topics and vibes every year. If you need some inspiration, consider any of these:
+
+- Have a tricky situation or question burning in you? Whether that's about language, technology, workplace dynamics, job hunting, etc., there's definitely others with the same issues in mind.
+- Made something recently you'd love to share? Whether that's requesting critique or offering a tool to help your fellow documentarians, we welcome you to bring your laptop and offer a demo.
+- Focused on building community? Some attendees put on recurring  sessions for LGBTQ+ groups, for neurodiverse groups, for job hunters, for folks in particular geographic regions, etc. If you've got a drive to set something up, we welcome that! 
+
+And as always, sign-ups are welcome during the conference. 
+
+## Monday Lightning Talk Submissions Open
+
+We're also opening up Monday Lightning Talk submissions. Lightning Talks are a wonderful way to share an idea, concept, or piece of information you find interesting, in an informal five-minute talk. Similar to above, we hope that by opening up Lightning Talks early, this provides more time to ideate and to submit your talk before the conference. [Learn more about giving a Lightning Talk here.](https://www.writethedocs.org/conf/portland/2025/lightning-talks/)
+
+<p style="margin: 2em 0;">
+<table border="0" cellpadding="0" cellspacing="0" style="background-color:#2ECC71; border:1px solid #4a4a4a; border-radius:5px;">
+<tr>
+  <td align="center" valign="middle" style="color:#FFFFFF; font-family:Helvetica, Arial, sans-serif; font-size:16px; font-weight:bold; letter-spacing:-.5px; line-height:150%; padding-top:15px; padding-right:30px; padding-bottom:15px; padding-left:30px;">
+     <a href="https://docs.google.com/forms/d/e/1FAIpQLSc9Op0AImTVbHoX4yi_AViNpCvE_Id-E_Q2-y3THQ7NxXW5MA/viewform?usp=sharing" target="_blank" style="color:#FFFFFF; text-decoration:none; border-bottom: none;">Submit your Monday Lightning Talk</a>
+  </td>
+</tr>
+</table>
+</p>
+
+```{figure} /_static/conf/images/headers/portland-2024-all-speakers.jpg
+```
+
+## Join our Slack Community
+
```

Comment:
Do we want to mention virtual at some point here?

Reply:
We mention virtual streaming in the FAQs; is there another place you want us to put something in?

Reply:
yeah, I mean just so folks attending virtually don't feel like an FAQ :japanese_ogre:

Reply:
Added!